docs/src/main/resources/news/2018-07-26-first-release.md

```diff
@@ -0,0 +1,113 @@
+
+* Device and Software Repository
+* Update Management
+* Artifact Content Delivery
+* Rollout Management
+
+which are accessible via the following interfaces:
+
+* Management UI
+* Management API
+* Direct Device Integration (DDI) API
+* Device Management Federation (DMF) API
+
+![hawkBit Overview](hawkBit_overview.jpeg)
+
+
+## What's new?
+
+Whenever there is a new release, the first question that comes to mind is: What's new? Since this is our first release, 
+one could argue that everything is new. However, most of the features are already well established. This holds true, for 
+example, for our APIs or the Rollout Management. Nevertheless, there have been some recent updates to hawkBit, which we 
+do not want to leave unmentioned: 
+
+### Streamlined UI
+
+The probably most noticeable change has been the removal of the two buttons (`Drop here to delete` and `Actions`) at the 
+bottom of the _Deployment_, _Distributions_, and _Upload_ view. This is a major usability improvement! For example, 
+deleting an item required (1) dragging an item onto the delete button, (2) opening the delete pop-up, and (3) confirming 
+the deletion. Now, an item can be easily removed by clicking on its remove icon and confirming the action. This is not 
+only faster and more intuitive, it also saves a lot of display real estate which can now be used to focus on what is 
+important. We hope you like this change as much as we do! _(Requires: hawkBit > 0.2.2)_
+
+![Screenshot of improved UI](hawkbit_ui.png)
+
+### MS SQL Server
+
+Eclipse hawkBit supports a range of different SQL databases. Up to now these were the internal H2 database (which can be 
+used for testing, development, or trial) and MySQL/MariaDB for production grade usage. This list is now extended by 
+Microsoft's SQL Server which is also available in production grade, as well as, IBM's DB2 for testing and development.
+
+### Docker images
+
+In order to enable interested parties to get started with hawkBit convenitely, we decided to provide the 
+[Update Server as a Docker image](https://hub.docker.com/r/hawkbit/hawkbit-update-server/) on Docker Hub. The image comes 
+in two flavors: the default image uses the internal H2 database, while the images with `-mysql` suffix contain the MySQL 
+driver to allow connecting a MySQL database. In addition to the Docker image, the hawkBit repository contains a 
+[docker-compose.yml](https://github.com/eclipse/hawkbit/blob/master/hawkbit-runtime/hawkbit-update-server/docker/docker-compose.yml) 
+that not only starts the Update Server, but futher includes a MySQL database and a RabbitMQ message queue. 
+
+To start the hawkBit Update Server image, open a terminal and run: 
+
+```
+$ docker run -d -p 8080:8080 hawkbit/hawkbit-update-server
+```
+> _Note: This requires a running [Docker deamon](https://docs.docker.com/install/) on your system._
+
+Now, browse to [http://localhost:8080](http://localhost:8080) and log-in with `admin:admin` to 
```

Comment:
> log-in with `admin:admin` to

Is there something missing after the last `to`?

Reply:
done